web/client/src/components/validation/TokenValidation.jsx

```diff
@@ -206,12 +196,12 @@ export default function ValidateToken() {
               {/* Box where token info will appear if users chooses to display it. */}
               {infoVisable && (
                 <div className="validation-message-div">
+                  <Typography variant="h5">Token Info</Typography>
```

Comment:
Rename to Token info for consistency across labels

Reply:
fixed!